clog/metrics_reporter.py

```diff
@@ -40,6 +41,8 @@ def record(self, *args, **kwargs):
 METRICS_TOTAL_PREFIX = METRICS_PREFIX + 'total.'
 LOG_LINE_SENT = 'log_line.sent'
 LOG_LINE_LATENCY = 'log_line.latency_microseconds'
+LOG_LINE_MONK_EXCEPTION = 'log_line.monk_exception'
+LOG_LINE_MONK_TIMEOUT = 'log_line.monk_timeout'
 
```

Comment:
Would it make sense to also add a metric for the size of the request?

Reply:
I'd like to add only the strictly needed metrics here, since the meteorite pipeline is not very fast (the latency is more or less equivalent to a scribe/monk message). I think the leaf is a better place where we can calculate this kind of stats.